build.fsx

```diff
@@ -59,6 +59,34 @@ Target "Docker" (fun _ ->
         info.Arguments <- "build -t cache ."
         info.WorkingDirectory <- ".") (System.TimeSpan.FromMinutes 5.0) |> ignore
 )
+
+Target "Migrations" (fun _ ->
+    ExecProcess (fun info ->
```

Comment:
Oooh nice, I was actually going to look into adding an iterator instead, Ill def make that change